systest/tests/distributed_post_verification_test.go

```diff
@@ -280,12 +281,12 @@ func TestPostMalfeasanceProof(t *testing.T) {
 		require.Equal(t, atx.Signature, invalidAtx.Signature)
 		require.Equal(t, atx.Coinbase, invalidAtx.Coinbase)
 		require.Equal(t, *atx.CommitmentATX, *invalidAtx.CommitmentATX)
-		require.Equal(t, atx.NIPostChallenge, invalidAtx.NIPostChallenge)
+		require.Equal(t, atx.NIPostChallenge, invalidAtx.NIPostChallengeV1)
 		require.Equal(t, atx.NIPost.Post.Indices, invalidAtx.NIPost.Post.Indices)
 
 		meta := &shared.ProofMetadata{
-			NodeId:          invalidAtx.NodeID.Bytes(),
-			CommitmentAtxId: invalidAtx.CommitmentATX.Bytes(),
+			NodeId:          invalidAtx.NodeID[:],
+			CommitmentAtxId: invalidAtx.CommitmentATX[:],
```

Comment:
Why the change? I know functionally there is no difference, but I think `.Bytes()` is easier to understand (for someone not familiar with the code).

Reply:
reverted